scripts/BuildAFComparisonTable.wdl

```diff
@@ -0,0 +1,289 @@
+version 1.0
+
+workflow BuildAFComparisonTable {
+	input {
+		File thouandG_vcf
```

Comment:
potential typo?
```suggestion
		File thousandG_vcf
```

Reply:
This is a copy of the actual workflow used to build the references, so I don't necessarily want to modify the code.

Reply:
I actually would suggest modifying the code slightly, for these types of typos and some similar things below.  This workflow contains a bunch of references to random jars stored in my google bucket that contain changes now merged into gatk.  So only a subset of broad folks would be able to run these workflows as they currently are anyway.  And for the sake of documentation, this type of typo fix doesn't change the understanding of how the files were generated, imo.